JavaScript/JavaScriptSDK.Tests/CheckinTests/Sender.tests.ts

```diff
@@ -1,9 +1,9 @@
 ﻿/// <reference path="..\TestFramework\Common.ts" />
 /// <reference path="../../JavaScriptSDK/Util.ts"/>
-/// <reference path="../../JavaScriptSDK/sender.ts" />
+/// <reference path="../../JavaScriptSDK/Sender.ts" />
 /// <reference path="../../JavaScriptSDK/SendBuffer.ts"/>
-/// <reference path="../../javascriptsdk/appinsights.ts" />
-/// <reference path="../../JavaScriptSDK/util.ts" />
+/// <reference path="../../javascriptsdk/Appinsights.ts" />
```

Comment:
All paths in source maps are case sensitive (at least in Edge). I'm trying to fix all casing issues, but I missed a couple spots.